src/index.d.ts

```diff
@@ -46,7 +46,7 @@ type Logger = {
 
 interface UserListQuery {
   hasNext?: boolean;
-  next(): Promise<User[]>;
+  next(): Promise<Array<User>>;
```

Comment:
Can we add this to: (https://github.com/sendbird/sendbird-uikit-react/blob/main/scripts/index_d_ts) ?

Reply:
Oh, I always forgot, thanks